### PR TITLE
Updated Create Alerts Step 1 Workbook 

### PR DESCRIPTION
**Pull request by M2skills**
## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

Moved Provider Instance dropdown outside the current dropdown group.
Made the Provider instance dropdown conditionally visible, hide the dropdown when provider type is HA Cluster.
Updated Headers above the dropdowns.

Alerts Page for HA Alerts 
For HA Alerts

Alerts Page For other Provider Types
For other Alerts

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook [hunk 1]
```diff
@@ -443,11 +424,59 @@
...
+        ],
+        "style": "above",
+        "queryType": 12
+      },
+      "conditionalVisibility": {
+        "parameterName": "ProviderType",
+        "comparison": "isNotEqualTo",
+        "value": "PrometheusHaCluster"
+      },
+      "name": "parameters - 16"
```
Comment (adityaku-ms) [inline Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook:478]: Please update the name here also

**Review by adityaku-ms — CHANGES_REQUESTED** [review]

File: Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook [hunk 2]
```diff
@@ -379,7 +379,7 @@
     {
       "type": 1,
       "content": {
-        "json": "### Choose a name, provider instance and severity for the alert"
+        "json": "### Choose a name, severity for the alert"
```
[the remark this answers is not in the document]
Comment (M2skills) [reply Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook]: updated name.

File: Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook (hunk 1 above)
Comment (M2skills) [reply Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook]: updated name.

File: Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook [hunk 3]
```diff
@@ -443,11 +424,59 @@
...
+      "type": 1,
+      "content": {
+        "json": "### Choose a provider instance for the alert"
+      },
+      "conditionalVisibility": {
+        "parameterName": "ProviderType",
+        "comparison": "isNotEqualTo",
+        "value": "PrometheusHaCluster"
+      },
+      "name": "text - 5 - Copy",
```
[the remark this answers is not in the document]
Comment (M2skills) [reply Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook]: updated name.

**Pull request closed without merging**